Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks-cartons.yml

```diff
@@ -193,3 +193,18 @@
           Quantity: 100
   - type: Sprite
     sprite: Objects/Consumable/Drinks/oatmilk.rsi
+    
+- type: entity
+  parent: [DrinkCartonVisualsOpenable, DrinkCartonBaseFull]
+  id: DrinkJuiceLemonCarton
+  name: lemon juice
+  description: First it's sour, then it's still sour.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        reagents:
+        - ReagentId: JuiceLemon
+          Quantity: 50
+  - type: Sprite
+    sprite: Objects/Consumable/Drinks/lemonjuice.rsi
```

Comment:
isnt this the same as DrinkLemonJuice but with a better name? Either move the description there and remove this one or make this one use a new sprite that its more like a carton and rename it to "lemon juice carton"

Reply:
there was 2 lemon juice glasses.  with the metamorphic glass update, there wasn't a need for that anymore so I removed the useless one and replaced it with the lemon juice carton.  The carton is a seperate entity from the drink version of it, so the carton has a different description.  Now its just 2 entities.  The juice carton, and the juice when it's in a glass.